### PR TITLE
Add contact email to readme

### PR DESCRIPTION
**Pull request by katjam**

### CODE REVIEW

**Comment by Basker12** [comment]
Hi

**Comment by Hiddenmaask** [comment]
which email?

File: README.md [hunk 1]
```diff
@@ -22,3 +22,7 @@ This repo contains exercises to use, as well as a Starting Package for the first
 * A meeting agenda to use
 * Exercises to use
 * A meeting to run a retrospective after the clubs
+
+## Get in touch
+
```
Comment (Basker12) [inline README.md:27]: I think you should include your name here so that people know this is your code :)

File: README.md [hunk 2]
```diff
@@ -22,3 +22,7 @@ This repo contains exercises to use, as well as a Starting Package for the first
...
+hello@code-reading.org
```
Comment (katjam) [inline README.md:28]:
```suggestion
email: hello@code-reading.org
```

**Pull request merged**